gravitee-apim-console-webui/src/management/api/documentation-v4/components/api-documentation-v4-newt-ai-button/api-documentation-v4-newt-ai-button.component.html

```diff
@@ -0,0 +1,23 @@
+<!--
+
+    Copyright (C) 2015 The Gravitee team (http://gravitee.io)
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+            http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+-->
+<div *gioPermission="{ anyOf: ['api-documentation-c'] }">
```

Comment:
optional feedback ;) 
but I would expect to see this permission condition more on the parent component